Automattic/jetpack
# My Jetpack: Update licensing feature flag default

Once #24395 is in, we should be in a state where the licensing UI in My Jetpack is ready for production. We now simply need to enable the licensing UI.

#### Changes proposed in this Pull Request:

* Updates the feature that controlled displaying the licensing UI in My Jetpack to default to true.

#### Jetpack product discussion



NA

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Checkout branch
* Install Jetpack
* Go to Jetpack Dashboard > My Plan
* Click "Activate a product" button and ensure that you land in licensing UI in My Jetpack
* Go to My Jetpack dashboard
* Ensure that "Activate a license" link shows in the "Your Plan" section
* Set `add_filter( 'jetpack_my_jetpack_should_enable_add_license_screen', '__return_false' );`
* Go to Jetpack Dashboard > My Plan
* Click "Activate a product" button and ensure that you stay in the main Jetpack dashboard
* Go to My Jetpack dashboard
* Ensure that you do not see a link to "Activate a license"
